travis-ymls/maxmin.travis.yml

```diff
@@ -0,0 +1,8 @@
+language: node_js
+arch:
+  - amd64
+  - ppc64le
+node_js:
+  - '14'
+#   - '12'
```

Comment:
are the builds for these versions failing?

Reply:
fixed typo